### DISCUSSION TITLE
Display version for custom integrations

### DISCUSSION THREAD

**Discussion by BottlecapDave** [home-assistant/frontend#20650] [Configuration] [answered]
It would be very useful if when clicking into a custom integration it could display the version from the integration's manifest. This would make support around custom integrations easier when querying user's if they're on the latest version, as depending on their knowledge and how they installed the integration, they might not realise they're on an old version.

<img width="1440" alt="Capture" src="https://github.com/home-assistant/frontend/assets/14993950/7a929939-42ee-4ee9-8708-6ec24d0c7ac5">

**Comment by karwosts** [comment, marked as answer]
Added by #20652. Thanks for the suggestion!
